crates/meilisearch/src/lib.rs

```diff
@@ -500,7 +501,20 @@ fn import_dump(
     let network = dump_reader.network()?.cloned().unwrap_or_default();
     index_scheduler.put_network(network)?;
 
-    let indexer_config = index_scheduler.indexer_config();
+    // 3.1 Use all cpus to process dump if `max_indexing_threads` not configured
+    let backup_config;
+    let base_config = index_scheduler.indexer_config();
+
+    let indexer_config = if base_config.max_threads.is_none() {
+        let thread_pool =
+            ThreadPoolNoAbortBuilder::new_for_indexing().num_threads(num_cpus::get()).build()?;
```

Comment:
Only use one thread in test suits me better:
```suggestion
        let (thread_pool, _) = default_thread_pool_and_threads()?;
```